amplify/backend/function/getUserReputation/src/index.js

```diff
@@ -30,11 +35,16 @@ exports.handler = async (event) => {
 
   const { skillId } = await colonyClient.getDomain(domainId ?? ROOT_DOMAIN_ID);
 
-  const { reputationAmount } = await colonyClient.getReputationWithoutProofs(
-    skillId,
-    walletAddress,
-    rootHash,
-  );
+  try {
+    const { reputationAmount } = await colonyClient.getReputationWithoutProofs(
+      skillId,
+      walletAddress,
+      rootHash,
+    );
+
+    return reputationAmount.toString();
+    // eslint-disable-next-line no-empty
+  } catch {}
 
-  return reputationAmount.toString();
+  return null;
```

Comment:
I don't think this disable is required. I think you can just catch the error, but not do anything with the variable

```suggestion
} catch (error) {
  return null;
}
```

But it works either way, so I don't really care. Leave it as it is.